tubular/scripts/frontend_utils.py

```diff
@@ -127,6 +153,20 @@ def create_version_file(self):
         except IOError:
             self.FAIL(1, 'Could not write to version file for app {}.'.format(self.app_name))
 
+    def copy_js_config_file_to_app_root(self, app_config, app_name):
+        """ Creates a copy of env.config.js(x) file to the app root """
+        source = app_config.get('JS_CONFIG_FILEPATH')
+        filename = app_config.get('LOCAL_CONFIG_FILENAME', "env.config.js")
+        # Skip if JS_CONFIG_FILEPATH not defined
+        if source:
```

Comment:
nit: it might be slightly more readable/concise if you fail early when `source` is not defined, so the "happy path" logic doesn't get nested under a conditional block.

```python
source = app_config.get('JS_CONFIG_FILEPATH')

# Skip if JS_CONFIG_FILEPATH not defined
if not source:
  return

filename = app_config.get('LOCAL_CONFIG_FILENAME', "env.config.js")
destination = f"{app_name}/{filename}"
try:
  shutil.copyfile(source, destination)
except FileNotFoundError:
  self.FAIL(1, f"Could not find '{source}' for copying for app {app_name}.")
except OSError:
  self.FAIL(1, f"Could not copy '{source}' to '{destination}', due to destination not writable.")
```

Reply:
Done